test/tftp/tftp_api_test.rb

```diff
@@ -17,6 +17,17 @@ def setup
     @args = { :pxeconfig => "foo" }
   end
 
+  def test_instantiate_syslinux
+    obj = app.helpers.instantiate "syslinux", "AA:BB:CC:DD:EE:FF"
```

Comment:
Is it really necessary? There is another PR https://github.com/theforeman/smart-proxy/pull/414 which will refactor all our tests and add more coverage for all available variants. I see the main purpose of the two tests to check for valid and invalid case. The rest will be covered by the others. Also this is a security fix, I'd prefer smallest possible patch.